lang/csharp/src/apache/main/Reflect/ReflectDefaultReader.cs

```diff
@@ -417,7 +417,7 @@ protected override object ReadRecord(object reuse, RecordSchema writerSchema, Sc
                     Field rf;
                     if (rs.TryGetField(wf.Name, out rf))
                     {
-//                        obj = _classCache.GetClass(writerSchema).GetValue(rec, rf);
+                        //                        obj = _classCache.GetClass(writerSchema).GetValue(rec, rf);
```

Comment:
delete the white space between `//` and ` obj` ?

Reply:
Fixed.